src/engine/instrument.h

```diff
@@ -310,6 +334,7 @@ struct DivInstrumentSTD {
   DivInstrumentMacro ex6Macro;
   DivInstrumentMacro ex7Macro;
   DivInstrumentMacro ex8Macro;
+  std::vector<DivInstrumentXattr> xattrs;
```

Comment:
Beware - this will totally break undo!

Reply:
How would I go around implementing undo for this?

Reply:
Making it part of struct DivInstrument rather than DivInstrumentSTD.
This also involves changing the record/playback undo/redo functions to take Xattr into account.